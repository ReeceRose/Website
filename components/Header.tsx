import Link from 'next/link';

const Header = (): JSX.Element => {
  return (
    <header>
      <div className="flex h-24 m-3 align-items-center">
        <Link href="/" className="font-bold">
          Reece Rose
        </Link>
        <nav className="ml-auto">
          <a
            href="mailto:reecerose18@gmail.com"
            target="_blank"
            rel="noopener noreferrer"
          >
            Contact
          </a>
        </nav>
      </div>
    </header>
  );
};

export default Header;
