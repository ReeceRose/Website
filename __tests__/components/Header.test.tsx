import { render } from '@testing-library/react';

import Header from '../../components/Header';

test('Header component matches snapshot', () => {
  const { asFragment } = render(<Header />);
  expect(asFragment()).toMatchSnapshot();
});
