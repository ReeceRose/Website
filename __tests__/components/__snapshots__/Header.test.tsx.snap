// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component matches snapshot 1`] = `
<DocumentFragment>
  <header>
    <div
      class="flex h-24 m-3 align-items-center"
    >
      <a
        href="/"
      >
        Reece Rose
      </a>
      <nav
        class="ml-auto"
      >
        <a
          href="mailto:reecerose18@gmail.com"
          rel="noopener noreferrer"
          target="_blank"
        >
          Contact
        </a>
      </nav>
    </div>
  </header>
</DocumentFragment>
`;
